utils/model_uploader/model_autotracing.py

```diff
@@ -0,0 +1,597 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+# This program is run by "Model Auto-tracing & Uploading" workflow
+# (See model_uploader.yml) to perform model auto-tracing and prepare
+# files for uploading to OpenSearch model hub.
+
+import argparse
+import json
+import os
+import shutil
+import sys
+import warnings
+from typing import List, Optional, Tuple
+from zipfile import ZipFile
+
+import numpy as np
+from mdutils.fileutils import MarkDownFile
+from numpy.typing import DTypeLike
+from sentence_transformers import SentenceTransformer
+
+# We need to append ROOT_DIR path so that we can import
+# OPENSEARCH_TEST_CLIENT and opensearch_py_ml since this
+# python script is not in the root directory.
+THIS_DIR = os.path.dirname(__file__)
+ROOT_DIR = os.path.join(THIS_DIR, "../..")
+sys.path.append(ROOT_DIR)
+
+LICENSE_PATH = "LICENSE"
+from opensearch_py_ml.ml_commons import MLCommonClient
+from opensearch_py_ml.ml_models.sentencetransformermodel import SentenceTransformerModel
+from tests import OPENSEARCH_TEST_CLIENT
+
+BOTH_FORMAT = "BOTH"
+TORCH_SCRIPT_FORMAT = "TORCH_SCRIPT"
+ONNX_FORMAT = "ONNX"
+
+TEMP_MODEL_PATH = "temp_model_path"
+ORIGINAL_FOLDER_PATH = "sentence-transformers-original/"
+TORCHSCRIPT_FOLDER_PATH = "sentence-transformers-torchscript/"
+ONNX_FOLDER_PATH = "sentence-transformers-onnx/"
+UPLOAD_FOLDER_PATH = "upload/"
```

Comment:
I assume we need to change this value?

Reply:
No, this is to upload artifact. It will be use temporarily for one github action job.